article/conf/routes

```diff
@@ -5,9 +5,9 @@
 # For dev machines
 GET     /assets/*path               dev.DevAssetsController.at(path)
 
-GET     /_healthcheck               conf.HealthCheck.healthcheck()
+GET     /_healthcheck               conf.HealthCheck.healthCheck()
 
-GET     /_cdn_healthcheck           controllers.CdnHealthcheckController.healthcheck()
+GET     /_cdn_healthcheck           conf.HealthCheck.healthCheck()
```

Comment:
Since we cache the internal healthcheck results, we can use the same method for normal and cdn healthcheck. Eventually we can get rid of the /_cdn_healthcheck endpoint